sidebars.js

```diff
@@ -164,9 +164,11 @@ module.exports = {
           label: "C++",
           items: [
             "languages/cpp/index",
+            "languages/cpp/authoring/index",
+            "languages/cpp/solution_template",
             {
               type: "category",
-              label: "igloo",
+              label: "Testing Framework",
```

Comment:
Let's do the following instead

1. Use label "Igloo" ("Testing Framework" sounds awkward)
2. Move `languages/cpp/igloo` to `languages/cpp/igloo/index` with `slug: /languages/cpp/igloo` and `sidebar_label: Overview`.

Reply:
Done.